pandapower/converter/cim/cim_tools.py

```diff
@@ -95,14 +95,15 @@ def extend_pp_net_cim(net: pandapowerNet, override: bool = True) -> pandapowerNe
 
     fill_dict['trafo'] = dict()
     fill_dict['trafo'][np_str_type] = [sc['t_hv'], sc['t_lv'], sc['pte_id_hv'], sc['pte_id_lv'], sc['tc'], sc['tc_id'],
-                                       'description', 'vector_group', 'id_characteristic']
+                                       'description', 'vector_group', 'id_characteristic_table',
+                                       'tap_dependency_table', 'tap_phase_shifter_type']
```

Comment:
should happen in create_empty_network, because its affecting not only CGMES converted networks
('id_characteristic' was special from CGMES, but its for all now)